corehq/couchapps/__init__.py

```diff
@@ -7,8 +7,16 @@
     'users_extra': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
     'all_docs': (
         None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta',
-        settings.NEW_DOMAINS_DB),
+        settings.NEW_DOMAINS_DB, settings.NEW_APPS_DB),
     'by_domain_doc_type_date': (
         None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta',
-        settings.NEW_DOMAINS_DB),
+        settings.NEW_DOMAINS_DB, settings.NEW_APPS_DB),
+
+    # register these views to both the main db AND the apps db.
+    'app_translations_by_popularity': (None, settings.NEW_APPS_DB),
+    'exports_forms_by_app': (None, settings.NEW_APPS_DB),
+    'forms_by_app_info': (None, settings.NEW_APPS_DB),
+
+    # register just to the new apps db
+    'cloudcare_apps': settings.NEW_APPS_DB,
```

Comment:
Ha, I'd forgotten about that guy (Mentally filed it away under general view cleanup, not *Application* view cleanup).  Yeah, I like that plan better - feels safer.